categorize_adjectives/keywords.py

```diff
@@ -0,0 +1,82 @@
+"""Categorize Adjectives by looking for keywords in definition"""
+import os
+from nltk.corpus import wordnet as wn
+
+# Folder where generated files are stored
+generated_file = os.path.join(os.getcwd(), '..', 'generated_files/')
+
+# Fetching all adjective synsets from Wordnet
+adjectives = wn.all_synsets('a')
+
+# Semantic Categories for adjectives
+categories = {'Perceptional' : [], 'Spatial' : [], 'Time' : [], 'Motion' : [], 'Substance' : [],
+              'Weather' : [], 'Body' : [], 'Mood' : [], 'Spirit' : [], 'Behavior' : [],
+              'Social' : [], 'Quantity' : [], 'Relational' : [], 'General' : [], 'Others' : []}
+
+# Keywords used to lookout for adjectives
+keywords = {}
+keywords['Perceptional'] = ['perceptional', 'lightness', 'color', 'sound', 'taste', 'smell',
+                            'surface']
+keywords['Spatial'] = ['spatial', 'dimension', 'direction', 'localization', 'origin', 'form',
+                       'existence']
+keywords['Time'] = ['time', 'velocity', 'age', 'habit']
+keywords['Motion'] = ['motion']
+keywords['Substance'] = ['substance', 'composition', 'state', 'stability', 'consistency',
+                         'ripeness', 'dampness', 'purity', 'gravity', 'weight', 'physics',
+                         'chemistry', 'temperature']
+keywords['Weather'] = ['weather', 'climate']
+keywords['Body'] = ['body', 'life', 'constitution', 'affliction', 'desire', 'sex', 'appearance',
+                    'bodily state']
+keywords['Mood'] = ['feeling', 'stimulus', 'mood']
+keywords['Spirit'] = ['spirit', 'intelligence', 'attention', 'knowledge', 'experience']
+keywords['Behavior'] = ['behavior', 'character', 'animal specific', 'skill', 'relation', 'sympathy',
+                        'inclination']
+keywords['Social'] = ['stratum', 'social', 'institution', 'poltics', 'religion', 'state', 'region']
+keywords['Quantity'] = ['quantity', 'number', 'cost', 'return']
+keywords['Relational'] = ['relational', 'validity', 'certainty', 'requirement', 'effectiveness',
+                          'difficulty', 'energy requirement', 'functioning', 'security', 'order',
+                          'linking', 'correspondence', 'accuracy', 'completeness', 'cause',
+                          'reference', 'beneficial effect']
+keywords['General'] = ['comparative', 'evaluation', 'standard']
+
```

Comment:
Please restructure the code below to 3 functions (finding adjectives, dumping the output, main)  and call them from main function.